scripts/devcontainer/post-create.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Upgrade NPM
+npm install --global npm@latest
+
+# Upgrade Pip
+pip install --upgrade pip
```

Comment:
When we add project dependencies I assume I can then update this to install requirements?

Reply:
Yes exactly! we'd just add another step, something like 

```bash
pip install --requirement path/to/requirements.txt
```